xbmc/interfaces/builtins/PVRBuiltins.cpp

```diff
@@ -74,6 +79,101 @@ static int SeekPercentage(const std::vector<std::string>& params)
   return 0;
 }
 
+namespace
+{
+/*! \brief Control PVR Guide window's EPG grid.
+ *  \param params The parameters
+ *  \details params[0] = Control to execute.
+ */
+int EpgGridControl(const std::vector<std::string>& params)
+{
+  CGUIWindowPVRGuideBase* guideWindow = nullptr;
+
+  if (params.empty())
+  {
+    CLog::Log(LOGERROR, "EpgGridControl(n) - No argument given");
```

Comment:
An earily return would make sense here, that way you could save one `if (guideWindow)` check further down

Reply:
yep, will change that